src/cuda.rs

```diff
@@ -4,6 +4,7 @@
 use super::core::*;
 use super::errors::*;
 use super::objdetect::{CSvmDetector, HogParams, ObjectDetect, SvmDetector};
+use ::*;
```

Comment:
Better not to import everything.

Reply:
Similar comments at other places with `use ::*`.

Reply:
I think it's ok because `::*` should contain very basic features such as `CVec`, `CResult` and others. They may not be enumerated one by one, I guess.